analyzer/src/main/kotlin/managers/Stack.kt

```diff
@@ -78,9 +78,12 @@ class Stack(
     override fun getVersionRequirement(): Requirement = Requirement.buildIvy("[2.1.1,)")
 
     override fun beforeResolution(definitionFiles: List<File>) =
+        // Stack could report version strings like:
+        // Version 1.7.1 x86_64
```

Comment:
Would it make sense to convert this comment into a unit test ?

Reply:
Not at this point, I believe. I just wanted to align with similar comments e.g at https://github.com/heremaps/oss-review-toolkit/blob/fd1c863b197f8eb0426057b1733c1cb27d735ecc/analyzer/src/main/kotlin/managers/PhpComposer.kt#L110-L111

If we want unit test coverage for these checks, we should do so consistently for all package managers.

Reply:
> If we want unit test coverage for these checks, we should do so consistently for all package managers.

This sentence is meaningless to me as it lacks an explanation ```why```, thus ignoring it for now.

Reply:
To me, "consistently" explained the "why": To be consistent. I don't think it makes much sense to add a unit test for this one case if you don't also cover the other similar cases, as it could lead to the false impression that this case is something special (which it is not).

Reply:
oh ok I see, I could not imagine that ```consistency``` was used as argument against writing a unit test as imo unit per definition cares only about the unit, not so much about other units. Anyhow, I'm fine with not doing the test now.